snmp/datadog_checks/snmp/data/default_profiles/ubiquiti-unifi.yaml

```diff
@@ -0,0 +1,66 @@
+extends:
+  - _base.yaml
```

Comment:
Looks like we're extending `_base.yaml` several times here (already extended by `_ubiquiti.yaml`)